777 fix filters

joemcelroy opened the pull request:
fixes #777

JasonStoltz commented on packages/search-ui-elasticsearch-connector/package.json:
```diff
@@ -41,6 +41,6 @@
     "@elastic/elasticsearch": "^8.2.1",
     "@elastic/search-ui": "1.15.0",
-    "@searchkit/sdk": "^3.0.0-canary.57"
+    "@searchkit/sdk": "^3.0.0-canary.58"
```
Why do you use the canary versions in SearchKit?

joemcelroy replied:
searchkit v3 not yet pass pre-release yet so still on canaries right now.

JasonStoltz reviewed (commented):
I see:
1) Code
2) Tests
3) Updated example

All looks well to me. Is there anything we need to note in our docs about this?

JasonStoltz reviewed (dismissed):
Hey Joe, I thought I added my review here but it looks like I did not. Everything looks good I'm just wondering if you need to update anything in docs?

joemcelroy commented:
> Hey Joe, I thought I added my review here but it looks like I did not. Everything looks good I'm just wondering if you need to update anything in docs?

Yeh its a good shout. I've added it to the connector api docs + another known issue. Hope these changes are ok now.
